cogito/protocol.go

```diff
@@ -212,18 +216,43 @@ func (src *Source) UnmarshalJSON(data []byte) error {
 // Validate verifies the Source configuration and applies defaults.
 func (src *Source) Validate() error {
 	//
-	// Validate mandatory fields.
+	// Evaluate mandatory fields.
 	//
 	var mandatory []string
-	if src.Owner == "" {
-		mandatory = append(mandatory, "owner")
+
+	defaultSinks := []string{"gchat", "github"}
+	defaultSinksSet := sets.From(defaultSinks...)
+	sinksSet := sets.From(src.Sinks...)
+
+	if sinksSet.Size() > 0 {
+		// First validate sinks are known and supported.
+		sinksNotValid := sinksSet.Difference(defaultSinksSet)
+		if sinksNotValid.Size() > 0 {
+			return fmt.Errorf("source: invalid sink: %s. Supported sinks: %s", sinksNotValid, defaultSinks)
```

Comment:
Plural: `invalid sinks:`. It can be more than one in my understanding.

Reply:
It could be, just for now we support only two, so it's rare to have more than one invalid.... I'll follow the suggestion.